docs/project_management/project_plan.adoc

```diff
@@ -37,10 +37,10 @@ Wir pflegen in unserem Team einen respektvollen Umgang miteinander. Alle Problem
 
 
 === Praktiken 
+
 - Iterative Entwicklung
 
-Zu Anfang jeder Iteration (7-tägig) werden Aufgaben an die einzelnen Teammitglieder vergeben, welche bis zur nächsten Iteration im angemessenen Maße bearbeitet werden sollen. Die Aufgaben sind normalerweise kleine Teilfunktionen der Webanwendung. 
-//funktionszuwachs??
+Zu Anfang jeder Iteration (7-tägig) werden Aufgaben an die einzelnen Teammitglieder vergeben, welche bis zur nächsten Iteration im angemessenen Maße bearbeitet werden sollen. Die Aufgaben sind kleine Teilfunktionen der Webanwendung. 
```

Comment:
Haben wir nicht auch Aufgaben, welche nicht Teil der Webanwendung sind? Iterationspläne, Tests, Use-Cases... Bin mir nicht sicher, ob wir das so schreiben sollten.

Reply:
soweit ich weiß, ist das eher ein Ist-Zustand. 
"Jedoch wird in einer Iteration auch der Iterationsplan erstellt und weitere Dokumente aktualisiert" würde ich da jetzt noch hinzufügen

Reply:
Was meinst du mit Ist-Zustand? Ist und Soll Zustand kenne ich aus dem Zusammenhang Anforderungsanalyse